src/testdir/test_functions.vim

```diff
@@ -1273,7 +1273,14 @@ endfunc
 func Test_executable_longname()
   CheckMSWindows
 
-  let fname = 'X' . repeat('あ', 200) . '.bat'
+  " Maximum length of a filename (including the path) in MS-Windows is 255
+  " characters. Make sure the length of the temp filename is < 255.
+  let len = 255 - getcwd()->len() - 6
```

Comment:
The maximum length of a pathname is 259, and the maximum length of a file name without the path is 255.